templates/zerver/help/enable-or-disable-pressing-enter-to-send.md

```diff
@@ -1,21 +1,20 @@
 # Press Enter to send
 
-By default, Zulip sends your message when you press enter (or return)
-on your keyboard. If you want to turn this off, simply uncheck the
-**Press Enter to send** box below the compose field:
+By default, Zulip sends your message when you press the **Send** button in
+the compose box. If you would like to send your message by pressing
+the enter (or return) key, follow these steps:
 
-![Box checked](/static/images/help/enter-box-checked.png)
+1. Check the **Press Enter to send** box below the compose field:
 
-You will now see a **Send** button below the compose field looking
-like this.
+  ![Box unchecked](/static/images/help/enter-box-unchecked.png)
 
-![Box unchecked](/static/images/help/enter-box-unchecked.png)
+2. Now press the enter (or return) key to send your message.
 
-With the box unchecked, you will have to click the **Send** button to
-send your message. Pressing enter will no longer send your message.
+!!! tip ""
```

Comment:
I generally don't like having tips one after another, so you could create a new section titled `## Disable pressing Enter to send` with this section under. If you do that, make sure to move the second tip before this section.